src/Core/MainLoop.js

```diff
@@ -142,7 +142,10 @@ MainLoop.prototype._renderView = function _renderView(view) {
         // use default rendering method
         this.gfxEngine.renderView(view);
     }
-
+    if (view._fullSizeDepthBuffer != null) {
```

Comment:
Why didn't you put this code in `GlobeView.preRender` ?

Reply:
because it will be erased if the user redefines `preRender`

Reply:
We already have this problem because we already do some stuff in Globe.preRender.

You should put depth buffer cache invalidation in preRender *and* add a todo/issue to state that this is a problem that should get fixed.

Reply:
updated